Presentation.Web/app/components/it-system/it-system-catalog.controller.ts

```diff
@@ -88,8 +104,54 @@
                         kendo.ui.progress(this.mainGrid.element, true);
                     });
                 }
+
             });
 
+            $scope.mySelectOptions = {
+                minimumInputLength: 1,
+                dropdownCss: { 'z-index': 100000, },
+                ajax: {
+                    data: (term, _) => {
+                        return { query: term };
```

Comment:
kan omskrives til  (term, _) => {query: term}
Det er generelt for alle one-statement lambdas at man kan bruge det trick